src/others/contexts/file-downloader.tsx

```diff
@@ -0,0 +1,138 @@
+import { createContext, useState, useContext, FunctionComponent, useEffect } from "react";
+import { decodeCategory, decodeLocation } from "../helpers/decode-aid-request";
+import { useAidRequestQuery, useLocationsQuery, useSuppliesQuery } from "./api";
+import { useFilter } from "./filter";
+
+type FormattedAidRequests = {
+  [city: string]: {
```

Comment:
**issue**

I don't think it's much needed to create yet another data shape. 

Note that here: https://github.com/ukraine-taskforce/requests-dashboard/pull/52 you have translation functions available - decodeCategory and decodeLocation is sub-optimal to say the least and should be deprecated. :)